pkg/trace/obfuscate/json.go

```diff
@@ -26,26 +27,42 @@ func (o *Obfuscator) obfuscateJSON(span *pb.Span, tag string, obfuscator *jsonOb
 }
 
 type jsonObfuscator struct {
-	keepers map[string]bool // these keys will not be obfuscated
+	keepKeys      map[string]bool // these keys will not be obfuscated
+	transformKeys map[string]bool // these keys pass through the transformer
```

Comment:
```suggestion
	keepKeys      map[string]bool // the values for these keys will not be obfuscated
	transformKeys map[string]bool // the values for these keys pass through the transformer
```
I guess this should help a bit?